Frontend/src/index.jsx

```diff
@@ -1,9 +1,42 @@
+// External Styles (this is probably not the best way to load this?)
+import '@thewca/wca-components/dist/index.esm.css'
+import React from 'react'
 import { createRoot } from 'react-dom/client'
-import App from './register/pages'
+import { createBrowserRouter, Link, RouterProvider } from 'react-router-dom'
+import Registrations from './pages/registrations'
+import Register from './pages/register'
+
+const router = createBrowserRouter([
```

Comment:
Putting it on the radar that as this piece grows, it should be split off into it's own file, maybe as "Router.js" or "Navigation.js"